packages/artifactor/test/contracts.js

```diff
@@ -13,15 +13,15 @@ const { Shims } = require("@truffle/compile-common");
 const tmp = require("tmp");
 tmp.setGracefulCleanup();
 
-describe("artifactor + require", () => {
+describe.skip("artifactor + require", () => {
   let Example, accounts, abi, bytecode, networkID, artifactor, config;
   const provider = Ganache.provider({
     miner: {
       instamine: "strict"
     }
   });
-  const web3 = new Web3();
-  web3.setProvider(provider);
+
+  const web3 = new Web3.Web3(provider);
```

Comment:
This shouldn't be necessary (@spacesailor24)

Reply:
Yes not anymore.

Reply:
If we want to use it in the same way we have to change the import to `const  { Web3 } = require("web3");` ,else I get the error `TypeError: Web3 is not a constructor`